### PR TITLE
Fix markup issues and warnings in iset.mm

### PR DESCRIPTION
**Pull request by digama0**
This one was already being markup checked, so the only change when adding metamath-knife checking is the annotations for metamath-knife to be able to parse the database. I added the garden_path stuff to @tirix's mathbox for consistency with set.mm.

### CODE REVIEW

**Review by jkingdon — APPROVED** [review]

**Review by benjub — APPROVED** [review]
LGTM. We should probably wait for @tirix's approval.

**Review by tirix — APPROVED** [review]
Now that I have a mathbox in `iset.mm`, I'll need to fill it up!

**Comment by jkingdon** [comment]
> Now that I have a mathbox in `iset.mm`, I'll need to fill it up!

Filling up iset.mm is always encouraged.

**Pull request merged**
